src/pcg/SimpleLevelGenerator.cpp

```diff
@@ -0,0 +1,22 @@
+#include "SimpleLevelGenerator.h"
+#include "SimpleRoomGenerator.h"
+
+void SimpleLevelGenerator::run(Scene &scene) {
+    float room_width = boundingBox.getWidth() / 4;
+    float room_height = boundingBox.getHeight() / 4;
+    float corridor_length = boundingBox.getWidth() / 3;
```

Comment:
What is going on with these magic numbers?

Reply:
One room has width equal to quarter of the level's width. Same with height.